docs/HowToGuides/GettingStarted.md

```diff
@@ -165,6 +165,7 @@ toolchain as a one-off, there are a couple of differences:
    * [Ubuntu 18.04](https://github.com/swiftlang/swift-docker/blob/main/swift-ci/main/ubuntu/18.04/Dockerfile)
    * [Ubuntu 20.04](https://github.com/swiftlang/swift-docker/blob/main/swift-ci/main/ubuntu/20.04/Dockerfile)
    * [Ubuntu 22.04](https://github.com/swiftlang/swift-docker/blob/main/swift-ci/main/ubuntu/22.04/Dockerfile)
+   * [Ubuntu 24.04](https://github.com/swiftlang/swift-docker/blob/main/swift-ci/main/ubuntu/24.04/Dockerfile)
```

Comment:
Do you reckon we could just link to https://github.com/swiftlang/swift-docker/tree/main/swift-ci/main to avoid maintaining this list?